browser/ui/brave_browser_command_controller.cc

```diff
@@ -141,10 +141,11 @@ void BraveBrowserCommandController::UpdateCommandForWebcompatReporter() {
 
 #if BUILDFLAG(ENABLE_TOR)
 void BraveBrowserCommandController::UpdateCommandForTor() {
-  const bool is_tor_enabled =
-      !brave::IsTorDisabledForProfile(browser_->profile());
-  UpdateCommandEnabled(IDC_NEW_TOR_CONNECTION_FOR_SITE, is_tor_enabled);
-  UpdateCommandEnabled(IDC_NEW_OFFTHERECORD_WINDOW_TOR, is_tor_enabled);
+  // Enable new tor connection only for tor profile.
+  UpdateCommandEnabled(IDC_NEW_TOR_CONNECTION_FOR_SITE,
+                       brave::IsTorProfile(browser_->profile()));
```

Comment:
Curious about this change? Looking at `profile_util.cc`, it makes sense that if the Tor window is already showing, it's overkill to check if service is disabled. But also, `IsTorDisabledForProfile` is checking if guest profile (which, I don't think it's a guest profile anymore, right?). Does this fix a bug?
cc: @yrliou

Reply:
I think this is fine because `New Tor Connection...` commands has only meaning to tor profile.
And command controller is per browser object.
Other normal/private guest browser doesn't use it. So, I think we don't need to enable this command for other profile.
With this change, app menu model can add this command to tor window's app menu w/o more explicit checking.

Reply:
For now, `IsTorDisabledForProfile` is used whether `New Tor Window` should be added or not(also for app menu). We want to add this command to only non-guest window.
So, both have different target window for adding.

Reply:
Makes sense! 😄👍